docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -77,6 +82,11 @@ in the message, like job ID, anomaly score, time, or top influencers.
 [role="screenshot"]
 image::images/ml-anomaly-alert-messages.jpg["Customizing your message"]
 
-After you save the configurations, the alert appears in the _Alerts and 
-Actions_ list where you can check its status and see the overview of its 
-configuration information.
+After you save the configurations, the alert appears in the *{alerts-ui}* list 
+where you can check its status and see the overview of its configuration 
+information.
+
+The name of an alert instance is always the same as the job ID of the associated 
+{anomaly-job}. You can mute the notifications for a particular {anomaly-job} on 
```

Comment:
The user can assign multiple anomaly detection jobs, hence we should adjust this sentence slightly